pkg/operator/assets/bindata.go

```diff
@@ -905,10 +905,12 @@ spec:
             fieldRef:
               fieldPath: metadata.namespace
         readinessProbe:
+          periodSeconds: 360
           httpGet:
             path: /readyz
             port: 8080
         livenessProbe:
+          periodSeconds: 360
```

Comment:
I have set `periodSeconds`(health check interval) equal to lease duration since that is the amount of time a new client will take to acquire leadership and get into ready state

Reply:
I don't think this is sufficient. It precludes running multiple pods, where the other pods are waiting to acquire leadership.

Reply:
Also, even if we were to go this route, I think that you would want to use `initialDelaySeconds` rather than `periodSeconds`.

Reply:
Do you think just `initialDelaySeconds` of 400 should be enough?

Reply:
No, that will have the same problem with multiple pods. Let's say there is one pod running that is the leader. Now, let's say that a second pod starts. The second pod should be allowed to run indefinitely waiting for the current leader to relinquish leadership. If the second pod is not serving to satisfy its liveness probe, then the kube controller will kill the pod rather than allowing it to continue waiting.

Reply:
So that means we should remove liveness probe then? because as per my understanding the second pod won't become leader (and continuously fail liveness probe) till the leader pod dies.

Reply:
No, that is not an acceptable solution either. The liveness probe is needed to ensure that a pod that has stopped working gets killed. The solution, as far as I see it, is to start serving the liveness prior to waiting to become the leader. The pod is alive even when not the leader.